ext/drsyms/libelftc/include/libdwarf.h

```diff
@@ -179,18 +179,18 @@ enum Dwarf_Form_Class {
 };
 
 #ifndef        DW_FRAME_HIGHEST_NORMAL_REGISTER
-#define        DW_FRAME_HIGHEST_NORMAL_REGISTER 63
+# define       DW_FRAME_HIGHEST_NORMAL_REGISTER 63
```

Comment:
IMHO this file shouldn't be changed: it's imported from libelftc and tweaking it is going to make it harder to diff.  (The ext/drsyms/libelftc dir should be excluded -- treated like third_party)